static/includes/General/CreateJiraSuggestion.md.part

```diff
@@ -8,6 +8,6 @@ To suggest a new feature, go to https://jira.ixsystems.com/projects/NAS/, log in
 
 ![JiraSuggestionCreate](/images/Contribute/JiraSuggestionCreate.png "Creating a new Suggestion")
 
-Briefly describe the new feature you'd like to see added in the **Summary** section.
+Briefly describe the new feature you would like to see added in the **Summary** section.
```

Comment:
The "here" link in line 2 does not work.
[here](https://jira.ixsystems.com/issues/?jql=issuetype%20%3D%20Suggestion%20AND%20status%20%3D%20%22Gathering%20Interest%22)